File: Exercises/Foundational/Intermediate-Level/3/fortran/E_O.f90
PROGRAM E_O
    IMPLICIT NONE
    REAL :: num
    CHARACTER(LEN=4) :: result  ! Declare result as a string of length 4 (for "even" or "odd")

    PRINT *, "Enter a number: "
    READ *, num

    ! Check if the integer part of the number is even or odd
    IF (MOD(INT(num), 2) == 0) THEN
        result = "even"
    ELSE
        result = "odd"
    END IF

    PRINT *, num, "is ", result
END PROGRAM E_O

File: Exercises/Foundational/Intermediate-Level/3/golang/E_O.go
package main;
import (
	"fmt"
)
func main(){
	var(
		result string
		num float64
	)
	fmt.Print("Enter a number: ")
	fmt.Scan(&num)
	if int(num)%2 == 0 {
		result = "even"
	} else {
		result = "odd"
	}

	fmt.Println(num,"is",result)
}